frontend/src/features/public-form/PublicFormPage.tsx

```diff
@@ -16,7 +16,7 @@ export const PublicFormPage = (): JSX.Element => {
       <FormStartPage />
       <PublicFormWrapper>
         <FormFields />
-        <FormEndPage />
+        <FormEndPage isPreview={false} />
```

Comment:
nit: probably don't need this prop, since the absence of a prop is automatically falsey.